docs/code/experimental/MlNetCookBook_StaticApi.md

```diff
@@ -0,0 +1,1026 @@
+# ML.NET Cookbook
+
+This document is intended to provide essential samples for common usage patterns of ML.NET, using examples from the experimental static API.
+The examples in this document make use of the static API, which at the time of writting is considered experimental.
+The supported way of composing pipelines in ML.NET is through the dynamic API. 
+You can find the same guide and examples on how to get started with ML.Net, dynamic API on the [cookbook reference for the dynamic API](..\MlNetCookBook.md)
+It is advisable to be at least minimally familiar with [high-level concepts of ML.NET](..\MlNetHighLevelConcepts.md), otherwise the terminology in this document may be foreign to you.
+We welcome feedback about the static API.  
```

Comment:
>. [](start = 40, length = 1)

let's add the motivation behind static API and how to use it. The stuff that I mentioned to be moved from the other md to here.